docs/running.md

```diff
@@ -9,14 +9,224 @@ a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet
 
 ## Kubernetes
```

Comment:
I think we should call this "Starting Kubernetes", and we should make sure the user has run `minikube start` or whatever the equivalent is for non-minikube setups.